petals-esb-deployer-api/src/main/resources/component-repository.xsd

```diff
@@ -29,6 +29,13 @@
                </documentation>
             </annotation>
          </element>
+         <element name="shared-library" type="cr:sharedLibrary" minOccurs="0" maxOccurs="unbounded">
```

Comment:
Perhaps can we use `xsd:all` instead of `xsd:sequence` to avoid to group components ans shared libraries writing a model ?

Reply:
A sequence of choice was used instead.